package.json

```diff
@@ -62,14 +63,13 @@
     "hsw-cli": "./bin/hsw-cli"
   },
   "scripts": {
-    "lint": "eslint $(cat .eslintfiles) || exit 0",
-    "lint-ci": "eslint $(cat .eslintfiles)",
+    "lint": "eslint $(cat .eslintfiles)",
     "lint-file": "eslint",
     "test": "bmocha --reporter spec test/*.js",
     "test-browser": "NODE_BACKEND=js bmocha --reporter spec test/*.js",
     "test-file": "bmocha --reporter spec",
     "test-file-browser": "NODE_BACKEND=js bmocha --reporter spec",
-    "test-ci": "istanbul cover --report lcovonly node_modules/.bin/bmocha -- --reporter spec test/*.js"
+    "test-ci": "node_modules/.bin/nyc --reporter lcov node_modules/.bin/bmocha -- --reporter spec test/*.js"
```

Comment:
Just a note, this can be abbreviated since `npm run ...` adds `node_modules/.bin/*` to the environment $PATH when running commands: https://docs.npmjs.com/cli/run-script 

but I agree enough bike shedding on this one for now